### PR TITLE
Drop model 

### PR DESCRIPTION
**Pull request by eddyxu**

### CODE REVIEW

File: tools/init.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+-- See images in rikai/tests/conftest.py
+CREATE EXTENSION IF NOT EXISTS pgtap;  -- For unit test
+
+CREATE TABLE images (
+image_id SERIAL PRIMARY KEY,
+image image
+);
+INSERT INTO images (image)
+VALUES
+(Row('http://farm2.staticflickr.com/1129/4726871278_4dd241a03a_z.jpg')),
```
Comment (changhiskhan) [inline tools/init.sh:20]: want to just pull these 2 images into the repo in case these get moved/deleted/updated?

File: Dockerfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 FROM postgres:14
 
 RUN apt-get update -y \
-	&& apt-get install -y -qq postgresql-plpython3-14 make python3-pip \
+	&& apt-get install -y -qq postgresql-plpython3-14 postgresql-14-pgtap make python3-pip \
```
Comment (changhiskhan) [inline Dockerfile:4]: Do we want this Dockerfile to also be for people to try it out? Should we even care to separate the test dependencies?

File: tools/init.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+psql <<EOF
+CREATE EXTENSION plpython3u;
+CREATE EXTENSION rikai;
+
+INSERT INTO ml.models (name, flavor, model_type)
```
Comment (changhiskhan) [inline tools/init.sh:7]: Is the plan to add a `CREATE MODEL` syntax like the other pg ml project has?

**Review by changhiskhan — APPROVED** [review]
some nits and questions

File: tools/init.sh (hunk 1 above)
Comment (eddyxu) [reply tools/init.sh:20]: good point
Filed #6

File: Dockerfile (hunk 2 above)
Comment (eddyxu) [reply Dockerfile:4]: Filed https://github.com/eto-ai/rikai-pg/issues/7

File: tools/init.sh (hunk 3 above)
Comment (eddyxu) [reply tools/init.sh:7]: It might not be.  `CREATE MODEL` requires changes in the parser. I have not figured out how to do it in postgres yet.

**Pull request merged**